ui/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/ProcessPosting.scala

```diff
@@ -19,9 +19,9 @@ class ProcessPosting {
     toRequest(ProcessConverter.toDisplayable(ProcessCanonizer.canonize(process), TestProcessingTypes.Streaming))
   }
 
-  def toEntityAsProcessToSave(process: EspProcess): RequestEntity = {
+  def toEntityAsProcessToSave(process: EspProcess, comment: String = ""): RequestEntity = {
```

Comment:
`comment` as `Option[String]`?

Reply:
It's required to pass comment in the request body, that's the reason it's mandatory here.